supervisor/docker/addon.py

```diff
@@ -130,11 +125,26 @@ def cgroups_rules(self) -> Optional[List[str]]:
                 device = self.sys_hardware.get_by_path(device_path)
             except HardwareNotFound:
                 _LOGGER.debug("Ignore static device path %s", device_path)
-            else:
-                rules.add(self.sys_hardware.policy.get_cgroups_rule(device))
+
+            # Check access
+            if not self.sys_hardware.policy.allowed_for_access(device):
+                _LOGGER.error(
+                    "Add-on %s try to access to blocked device %s!",
+                    self.addon.slug,
+                    device.name,
+                )
+                continue
+            rules.add(self.sys_hardware.policy.get_cgroups_rule(device))
 
         # Attach correct cgroups for devices
         for device in self.addon.devices:
+            if not self.sys_hardware.policy.allowed_for_access(device):
+                _LOGGER.error(
+                    "Add-on %s try to access to blocked device %s!",
+                    self.addon.slug,
```

Comment:
This will end up in user logs, the slug is fine for developers but can be misleading for the user.
This should be `self.addon.name` or even better `Add-on %s(%s)` and then `self.addon.name, self.addon.slug`

(same goes for all the other errors/warnings)